ragna/deploy/_ui/modal_configuration.py

```diff
@@ -198,8 +229,7 @@ async def model_section(self):
             ),
         )
 
-    @pn.depends("config", "config.assistant_name", "config.source_storage_name")
-    def advanced_config_ui(self):
+    def advanced_config_upload(self):
```

Comment:
Split into two functions (one for the upload tab, and another for the corpus tab), as a bunch of the options are not relevant for the corpus tab.